### ISSUE TITLE
resetday wrong?

### ISSUE THREAD

**Issue by jack206418070** [kakysha/HonorSpy#115]
https://github.com/kakysha/HonorSpy/blob/ba5cf143236b20e6fdeb29127b616e9c67261ebc/honorspy.lua#L528

resetday = 3 in Asia server  or not?          Taiwan resetDay is Thursday.

thx so much.

**Comment by jack206418070** [comment]
If you want to chage china"s resetTime, you should modify china"s regionId to chage resetDay = 1.

like
```
elseif (regionId == 3)
      resetDay = 1
      resetHour = 22
```


thx so much.

**Comment by kakysha** [comment]
What?
What are PvP reset day and time for China and Taiwan then? I was told that 
Korea, Taiwan, China: 10 PM UTC Mon (7 AM KST Tue)

**Comment by jack206418070** [comment]
Taiwan : 10 PM UTC Wed 
China :   10 PM UTC Mon
Korea:    I don't konw

**Comment by tzn521** [comment]
是的，之前我说错了。希望能够修改下，非常感谢！！台湾的更新时间我就不知道了。。可能台服就是如同这位所说的时间了！

**Comment by amotosouoc** [comment]
EU Region is 1, at least is what I get when I run GetCurrentRegion(). This causes the reset to happen on tuesday instead of wednesday.

**Comment by kakysha** [comment]
fixed in 1.7.2
